pyScienceMode2/sciencemode.py

```diff
@@ -101,18 +125,68 @@ def __init__(self, port: str, show_log: bool = False, with_motomed: bool = False
         self.__comparison_thread_started = False
         self.__watchdog_thread_started = False
         self.command_send = []  # Command sent to the rehastim
-        self.command_received = []  # Command received by the rehastim
-        self.Type = Type
+        self.ack_received = []  # Command received by the rehastim
+        self.TypeReha2 = TypeReha2
+        self.TypeRehap24 = TypeRehap24
         self.error_occured = (
             False  # If the stimulation is not working and error occured flag set to true, raise an error
         )
-
+        self.stimulation_active = True  # If the stimulation is active, set to true
```

Comment:
Why active by default ?

Reply:
Removed, it is True when we send the first InitAck to start the thread. it is used to stop the inital infinite loop of the thread ( the initial while 1)